samples/highcharts/demo/column-basic/demo.js

```diff
@@ -3,67 +3,45 @@ Highcharts.chart('container', {
         type: 'column'
     },
     title: {
-        text: 'Monthly Average Rainfall'
+        text: 'Corn vs wheat estimated production for 2020 (1000 MT)'
```

Comment:
It looks a bit crammed in mobile view, so I think we should
* Remove the "(1000 MT)" part. The unit is given in the y-axis title so we don't need it in the main title.
* Left-align the title and subtitle to make it appear more tidy.